src/main/java/org/tron/core/capsule/ReceiptCapsule.java

```diff
@@ -119,11 +131,10 @@ private void payEnergyBill(
       this.setEnergyUsage(usage);
     } else {
       energyProcessor.useEnergy(account, accountEnergyLeft, now);
-      long SUN_PER_ENERGY = manager.getDynamicPropertiesStore().getEnergyFee() == 0
-          ? Constant.SUN_PER_ENERGY
-          : manager.getDynamicPropertiesStore().getEnergyFee();
+      long dynamicEnergyFee = manager.getDynamicPropertiesStore().getEnergyFee();
+      long sunPerEnergy = dynamicEnergyFee > 0 ? dynamicEnergyFee: Constant.SUN_PER_ENERGY;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Convert this usage of the ternary operator to an "if"/"else" structure. ([squid:S1774](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1774))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbTWpNNLpqqSUlcalJx&open=AWbTWpNNLpqqSUlcalJx)

[ISSUE_KEY]: AWbTWpNNLpqqSUlcalJx